docs/json_schema_cheat_sheet/05-property-reuse.md

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
JSON Schema has defs/refs which can also do this. Should we mention why YAML anchors are better alternative?